web/docs/tql2/operators/decompress_brotli.md

```diff
@@ -0,0 +1,32 @@
+# decompress_brotli
+
+Decompresses a stream of bytes in the Bzip2 format.
+
+```tql
+decompress_brotli
+```
+
+## Description
+
+The `decompress_brotli` operator decompresses bytes in a pipeline incrementally.
+The operator supports decompressing multiple concatenated streams
+of the same codec transparently.
+
+:::note Streaming Decompression
+The operator uses [Apache Arrow's compression
+utilities][apache-arrow-compression] under the hood, and transparently supports
+all options that Apache Arrow supports for streaming decompression.
+:::
```

Comment:
Should we just delete this? I don't think anybody cares, plus we don't expose any options for `decompress` anyway.